app/colculater.py

```diff
@@ -0,0 +1,61 @@
+import math
+import datetime
+
+from app.customers import Customers
+
+
+def calculater(
+        customer: Customers,
+        shops: list,
+        fuel_price: float
+) -> None:
+    print(f"{customer.name} has {customer.money} dollars")
+
+    customer_location = customer.location
+    fuel_consumption = customer.car["fuel_consumption"]
+    total_value = 1_000_000
+    best_shop = None
+    product_price = 0
+
```

Comment:
Yes, here you can create a class named Calculator, which has several methods such as calculating the distance, calculating the consumption, printing methods etc.